do-release-tags

```diff
@@ -67,7 +93,21 @@ for (name,newcommit) in releasedata['releases'].iteritems():
                 print("No changes in {} = {}, promoted from {}".format(ref, current, currentpromoted))
                 continue
         print("Previously: {} = {}, promoted from {}".format(ref, current, currentpromoted))
-    _,newcommitdata,_ = r.load_commit(newcommit)
+    try:
+        _,newcommitdata,_ = r.load_commit(newcommit)
```

Comment:
I'd make this `[_,havecommit] = r.has_object(OSTree.ObjectType.COMMIT, newcommit); if havecommit:` to avoid catching things like I/O errors.